akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -139,6 +140,7 @@ abstract class ActorRef extends java.lang.Comparable[ActorRef] with Serializable
    * But you cannot rely on that it is alive if it returns false, since this by nature is a racy method.
    */
   @deprecated("Use context.watch(actor) and receive Terminated(actor)", "2.2")
+  @InternalApi
   private[akka] def isTerminated: Boolean
```

Comment:
we can remove the deprecated now, if that helps
then one more place in EmptyLocalActorRef